assets/lokomotive-kubernetes/bootkube/resources/charts/calico/templates/deployment.yaml

```diff
@@ -0,0 +1,124 @@
+apiVersion: apps/v1
```

Comment:
Could you put a comment on the source of the config, in case someone else were to update them?

Reply:
Done.